### PR TITLE
Add the project files.

### PR DESCRIPTION
**Pull request by Abe1able**
### Ethio Tech Conference - Final Capstone Project

This project, made as a first capstone project for Microverse's HTML, CSS, and Javascript module, is a website for an imaginary Global Tech Conference, to be held in Addis Ababa, Ethiopia. I've tried using the best practices for HTML, CSS, and javascript for this project. I've also made it responsive using media queries.

#### Here is a link to a video of me showcasing the project. 

https://www.loom.com/share/bc9b0460eba449168208f0836e534c36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Install all the dependencies by running the command npm -i
 
 ## Live Demo 
 
-[Demo](https://abe1able.github.io/Personal-Portfolio/)
+[Demo](https://abe1able.github.io/Ethio-Tech-Conference/index.html)
 
 ## Authors
```
Comment (franclobo) [inline README.md:35]: - [x] You almost have a professional README file, please 🙏🏽 replace `👤 **Author**` with your name.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+    <main class="main">
+      <section class="headline-section">
+        <div class="headline-title-text">
+          <p>"Hello! Tech Enthusiasts"</p>
+          <h2>
+            ETHIO GLOBAL TECH <br />
+            CONFERENCE <br /> 
+            2022
+          </h2>
```
Comment (franclobo) [inline index.html]: - [x] Kindly use `h1` tag for the principal topic of the page, for the best practices of HTML. 👍🏽

File: about.html [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    </header>
+
+    <main>
+      <section class="headline-section">
+        <div class="headline-title-text">
+          <h2>
+              ETHIO GLOBAL TECH <br />
+              CONFRENCE <br /> 
+              2022
+            </h2>
```
Comment (franclobo) [inline about.html]: - [x] Kindly use `h1` tag for the principal topic of the page, for the best practices of HTML. 👍🏽

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Abe1able, 

# STATUS: REQUIRED CHANGES

Good job so far!

I feel proud of your progress 👏🏽 there are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct GitHub flow ✔️ 
- Descriptive PR ✔️ 

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Kindly record again your video and mention some interesting piece of code or something you built that you are very proud of. 😊 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: about.html (hunk 3 above)
Comment (Abe1able) [reply about.html]: Fixed, Thank you.

File: index.html (hunk 2 above)
Comment (Abe1able) [reply index.html]: Fixed, Thank you.

File: README.md (hunk 1 above)
Comment (Abe1able) [reply README.md:35]: Fixed, Thank you

**Review by Qoosim — APPROVED** [review]
# *Status: Approved* :tada: 

Hi @Abe1able, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

Good luck! 🎉 

## Top Highlights

-  No linter errors. :heavy_check_mark:
-  Good README file. :heavy_check_mark:
-  Meaningul commit messages. :heavy_check_mark: 
-   HTML, CSS, && JS Best Practices. :heavy_check_mark: 




Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**